src/main/java/com/sap/sgs/phosphor/fosstars/nvd/data/CpeMatch.java

```diff
@@ -45,6 +53,16 @@ public void setCpe23Uri(String cpe23Uri) {
     this.cpe23Uri = cpe23Uri;
   }
 
+  @JsonProperty("versionEndExcluding")
+  public String getVersionEndExcluding() {
```

Comment:
Did I miss something here. Where are these being `set`?
- versionEndExcluding
- versionEndIncluding

Reply:
They are set during deserialization of JSON data from NVD.